pkg/storageos/deploy.go

```diff
@@ -17,9 +17,9 @@ const (
 	// IntreeProvisionerName is the name of the k8s native provisioner.
 	IntreeProvisionerName = "kubernetes.io/storageos"
 	// CSIProvisionerName is the name of the CSI provisioner.
-	CSIProvisionerName = "storageos"
+	StorageOSProvisionerName = "storageos"
 	// StorageOSProvisionerName is the new CSI provisioner name.
-	StorageOSProvisionerName = "csi.storageos.com"
+	CSIProvisionerName = "csi.storageos.com"
```

Comment:
The comments above the variables have been interchanged :slightly_smiling_face:

Reply:
Actually, the comments were still good, the values were swapped.  Fixed now.